### DISCUSSION TITLE
Error such as for no code action should use user-error instead

### DISCUSSION THREAD

**Discussion by Inc0n** [joaotavora/eglot#1053] [Ideas]
I use `(setq debug-on-error t)` on all times, and some times I do click onto the code with error messages, which trigger `eglot--error`, which can be cumbersome to deal with.
And so, if `user-error` could be used instead, it would be just a message in minibuffer.

**Comment by joaotavora** [comment]
I think this makes sense. Submit a PR. We'll need `eglot--user-error` probably

**Reply by Inc0n** [reply]
Right! So here it is: https://github.com/joaotavora/eglot/pull/1054. There are mostly stemming from utilization of mouse interaction.
